### ISSUE TITLE
update to chokidar v3

### ISSUE THREAD

**Issue by catdad** [catdad/electronmon#25]
this has a dependency on node 8+... not sure what that is in electron versions

**Comment by catdad** [comment]
Due to https://github.com/paulmillr/chokidar/issues/873, I'd have to wait for watching to start before starting the app. Consider watchboy instead to avoid slowing down startup, as well as the Windows/Linux improvements. #30

**Comment by catdad** [comment]
Closing in favor of #30
